src/maps.c

```diff
@@ -20,6 +20,7 @@ struct hash_elem *type_map = NULL;
 struct hash_elem *role_map = NULL;
 struct hash_elem *user_map = NULL;
 struct hash_elem *attr_map = NULL;
+struct hash_elem *roleattr_map = NULL;
```

Comment:
Is there a reason this is named "roleattr" when it's a map of "attribute_role"s rather than roleattributes?

Reply:
changed